histomicstk/cli/NucleiDetection/NucleiDetection.xml

```diff
@@ -218,4 +218,36 @@
       <default>1</default>
     </integer>
   </parameters>
+  <parameters advanced="true">
+    <label>Frame and Style</label>
+    <description>Frame parameters</description>
+    <string>
+      <name>frame</name>
+      <longflag>frame</longflag>
+      <label>Frame Index</label>
+      <description>Frame index in a multi-frame image</description>
+      <default>{#control:#current_image_frame#}</default>
+    </string>
+    <string>
+      <name>style</name>
+      <longflag>style</longflag>
+      <label>Style Options</label>
+      <description>Image style options for compositing a multi-frame image</description>
+      <default>{#control:#current_image_style#}</default>
+    </string>
+  </parameters>
+  <parameters advanced="true">
+    <label>Image inversion</label>
+    <description>Choose if image inversion is needed</description>
+    <string-enumeration>
+      <name>ImageInversionForm</name>
+      <description>Image inversion may be needed for greyscale images with bright nuclei and dark background. Choose if color inversion is needed.</description>
+      <channel>input</channel>
+      <longflag>output_form</longflag>
+      <element>Yes</element>
+      <element>No</element>
+      <element>unspecified</element>
```

Comment:
See my comments elsewhere -- I think the default should be `default` rather than `unspecified` with some logic based on number of bands.